cdap-docs/admin-manual/source/_includes/installation/configuration.txt

```diff
@@ -171,12 +171,29 @@ this set of directories are properties specified in the ``cdap-site.xml`` file,
 described in the :ref:`appendix-cdap-site.xml`):
 
 - ``app.temp.dir`` (default: ``/tmp``)
-- ``kafka.log.dir`` (default: ``/tmp/kafka-logs``)
+- ``kafka.server.log.dirs`` (default: ``/tmp/kafka-logs``)
+- ``local.data.dir`` (default: ``data``; if this is instead an absolute path, needs to be writable)
```

Comment:
This is also a strangely long list for "Temp Files".  I think we should rename this to "Local Storage Configuration" or similar.